plugin/pkg/scheduler/core/generic_scheduler.go

```diff
@@ -246,64 +245,69 @@ func findNodesThatFit(
 	metadataProducer algorithm.PredicateMetadataProducer,
 	ecache *EquivalenceCache,
 ) ([]*v1.Node, FailedPredicateMap, error) {
-	var filtered []*v1.Node
+
 	failedPredicateMap := FailedPredicateMap{}
+	filtered := nodes
 
-	if len(predicateFuncs) == 0 {
-		filtered = nodes
-	} else {
+	if len(predicateFuncs) > 0 {
 		// Create filtered list with enough space to avoid growing it
 		// and allow assigning.
-		filtered = make([]*v1.Node, len(nodes))
-		errs := errors.MessageCountMap{}
-		var predicateResultLock sync.Mutex
-		var filteredLen int32
+		filtered = make([]*v1.Node, 0, len(nodes))
 
 		// We can use the same metadata producer for all nodes.
 		meta := metadataProducer(pod, nodeNameToInfo)
+
+		errs := errors.MessageCountMap{}
+		var predicateResultLock sync.Mutex
+
 		checkNode := func(i int) {
 			nodeName := nodes[i].Name
+
```

Comment:
Please remove all the added blank lines. They are not improving readability.

Reply:
All blank lines? I feel sometimes adding blank lines make it look better, and help me sort things out.

Reply:
That's personal preference. Generally speaking, blank lines should be avoided for stylization and should be limited to cases that separation of pieces of code makes the logic easier to understand.